src/generic/datavgen.cpp

```diff
@@ -2501,13 +2653,15 @@ void wxDataViewMainWindow::OnPaint( wxPaintEvent &WXUNUSED(event) )
     }
 
 #if wxUSE_DRAG_AND_DROP
-    if (m_dropHint)
+    wxRect drop_item_rect;
```

Comment:
This code doesn't always use consistent naming convention, but it's still better to name new variables in wx standard `camelCase` rather than `snake_case`.